code/modules/mob/living/carbon/human/species/xenomorphs/alien_species.dm

```diff
@@ -300,7 +301,6 @@
 		)
 
 	inherent_verbs = list(
-		/mob/living/proc/ventcrawl,
```

Comment:
No, keep the vent crawl, the queen is already slow, and xenomorphs can just be that flexible.

Reply:
Alright. Will do.